tests/org.eclipse.passage.lic.emf.tests/src/org/eclipse/passage/lic/emf/meta/ComposedClassMetadataTest.java

```diff
@@ -0,0 +1,52 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.emf.meta;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+import java.util.Optional;
+
+import org.eclipse.emf.ecore.EClass;
+import org.eclipse.emf.ecore.EStructuralFeature;
+import org.eclipse.emf.ecore.EcoreFactory;
+import org.junit.Test;
+
+public class ComposedClassMetadataTest {
+
+	private final EClass type = EcoreFactory.eINSTANCE.createEClass();
+	private final EStructuralFeature id = EcoreFactory.eINSTANCE.createEAttribute();
+	private final EStructuralFeature name = EcoreFactory.eINSTANCE.createEReference();
+	private final EntityMetadata metadata = new PlainEntityMetadata(type, id, name);
+	private final ClassMetadata searcher = c -> Optional.of(metadata);
+
+	@Test(expected = NullPointerException.class)
+	public void testNullConsider() {
```

Comment:
`test` prefix for `@Test`-method is no longer mandatory, we can name tests in a less verbose way with more sence.

Reply:
wow! I'll consider it next time